assets/vue/App.vue

```diff
@@ -31,7 +31,11 @@
           </ul>
           <ul class="navbar-nav flex-row flex-wrap ms-md-auto" id="navbarAPI">
             <li class="nav-item">
-              <a class="nav-link" href="https://github.com/openSUSE/qem-dashboard/blob/main/API.md" target="_blank">
+              <a
+                class="nav-link"
+                href="https://github.com/openSUSE/qem-dashboard/blob/main/docs/API.md"
```

Comment:
This URL returns a 404 page.

Reply:
The linked .md file is currently still located in the root directory and is moved to the docs dir, fixing this link, as part of the PR